packages/lit-dev-content/site/tutorials/content/async-directive/04.md

```diff
@@ -0,0 +1,25 @@
+Instead of extending the basic `Directive` class, instead import and extend
+`AsyncDirective`.
+
+```ts
+import {AsyncDirective} from 'lit/async-directive.js';
```

Comment:
```suggestion
import {directive, AsyncDirective} from 'lit/async-directive.js';
```

and tell the user to remove the import ... from 'lit/directive.js'.... and fix the solve to do this.